docs/src/main/asciidoc/hibernate-orm-panache.adoc

```diff
@@ -156,6 +168,13 @@ columns as public fields:
 
 [source,java]
 ----
+package org.acme;
+
+import java.time.*;
+import java.util.*;
+import javax.persistence.*;
```

Comment:
We tend to avoid star imports. It would be nice to use only the classes needed in the snippet